app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people: dict[str, "Person"] = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    # Creating all Person instances
+    for person_dict in people:
+        Person(person_dict["name"], person_dict["age"])
+
+    # Seting the wife/husband attributes
+    for person_dict in people:
+        person_instance = Person.people[person_dict["name"]]
+        if "wife" in person_dict and person_dict["wife"] is not None:
```

Comment:
You can check if smth in dict this way -> `if preson_dict.get("wife", None): `
Instead of 2 check as you did 
Same on 20 line